repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/utils/TestUserUtils.scala

```diff
@@ -1,27 +1,25 @@
 package cromwell.pipeline.datastorage.dao.repository.utils
 
-import java.util.UUID
-
-import cromwell.pipeline.datastorage.dto.User.UserEmail
-import cromwell.pipeline.datastorage.dto.{ User, UserId }
+import cromwell.pipeline.datastorage.dto.{ Name, UUID, User, UserEmail }
+import cromwell.pipeline.datastorage.dto.auth.Password
 import cromwell.pipeline.utils.StringUtils
+import cats.implicits._
 
 object TestUserUtils {
-  val userPassword = "-Pa$$w0rd-"
+  val userPassword: Password = Password("-Pa$$w0rd-")
 
   def getDummyUser(
-    uuid: String = UUID.randomUUID().toString,
-    email: UserEmail = "JohnDoe-@cromwell.com",
+    uuid: UUID = UUID.random,
     password: String = userPassword,
     passwordSalt: String = "salt",
-    firstName: String = "FirstName",
-    lastName: String = "Lastname",
+    firstName: Name = Name("FirstName"),
+    lastName: Name = Name("LastName"),
     active: Boolean = true
   ): User = {
     val passwordHash = StringUtils.calculatePasswordHash(password, passwordSalt)
     User(
-      UserId(uuid),
-      s"JohnDoe-$uuid@cromwell.com",
+      uuid,
+      UserEmail(s"JohnDoe-${uuid.unwrap}@cromwell.com"),
```

Comment:
Something is wrong.
We should not be able to unsafely create an instance of `UserEmail` without `Unsafe` object.
@DenisVerkhoturov what is going on? :)

Reply:
My fault, fixed

Reply:
Done